docs/src/getting-started.rst

```diff
@@ -431,6 +488,44 @@ Define a stack and add it to the app.
 
 .. tabs::
 
+    .. group-tab:: C#
+
+        Create **MyStack.cs**:
+
+        .. code-block:: c#
+
+            using Amazon.CDK;
+
+            namespace HelloCdk
+            {
+                public class MyStack: Stack
+                {
+                    public MyStack(App parent, string name) : base(parent, name, new StackProps())
```

Comment:
I believe you should be able to pass in `null` since we fixed https://github.com/awslabs/jsii/issues/282 (and also, looking forward for you to address https://github.com/awslabs/jsii/issues/210).

Reply:
Looking now

Reply:
I'll see if this works.